src/widgets/gps-details-widget/widget.tsx

```diff
@@ -0,0 +1,48 @@
+import { Flex, Heading, View, StatusLight } from '@adobe/react-spectrum';
+import { BaseRendererProps } from '@wuespace/telestion-client-types';
+import { useChannelLatest } from '@wuespace/telestion-client-core';
+import { LoadingIndicator } from '@wuespace/telestion-client-common';
+
+import { GpsDataMessage } from '../../model/messages';
+import { GpsData } from '../../model/channels';
+import { DetailsTable } from './details-table';
+
+export function Widget({ title }: BaseRendererProps) {
+	const latestData = useChannelLatest<GpsDataMessage>(GpsData)?.result[0];
+
+	const isConnected = latestData && latestData.fix;
+
+	return (
+		<View width="100%" height="100%" padding="size-200">
+			<Flex direction="column" width="100%" height="100%">
+				<Flex
+					flexGrow={0}
+					direction="row"
+					justifyContent="space-between"
+					alignItems="start"
+				>
+					<Heading marginTop={0} flexGrow={0} level={3}>
+						{title}
+					</Heading>
+					<StatusLight
+						marginTop={-1}
+						variant={isConnected ? 'positive' : 'negative'}
+					>
+						{isConnected ? 'Connected' : 'Disconnected'}
```

Comment:
Can we find some more descriptive names here? Maybe something like _Fix established_ and _Low Precision_ or something like that? _Connected_/_Disconnected_ might be confused with the connection status of the client application (or backend application to some other device) 🤔

Reply:
How sounds _Located_ and _Interrupted_?

Reply:
Or _active_ and _suspended_?

Reply:
I think that's a good idea. Maybe later, we'll get some even better ideas, but for now, that sounds great 🙂

Reply:
_Discovered_ and _Lost?

Reply:
Applied as _Fix established_ and _Fix pending_ in https://github.com/TelestionTeam/telestion-rocketsound-psc/pull/75/commits/d2f9bbcea5ff7c640fc82d4e0b2798df23b06ee7 :wink: